cfme/tests/services/test_add_remove_vm_to_service.py

```diff
@@ -90,12 +92,16 @@ def add_to_service
         .methods.create(name='InspectMe', location='inline', script=method_torso)
 
     request.addfinalizer(method.delete_if_exists)
+    vm = VM.factory(random_vm_name(), provider, small_template_modscope.name)
+    vm.create_on_provider(find_in_cfme=True, timeout=700, allow_skip="default")
+    request.addfinalizer(vm.delete_from_provider)
```

Comment:
**REQUIRED**
I would move vm creation to a fixture.

Reply:
Okay let me try this one

Reply:
I'm not sure this is required since this file has only one test

Reply:
I believe that in the test we should only test, and other prerequisite stuff should moved to fixtures.